addons/mass_mailing_event/models/event_event.py

```diff
@@ -16,7 +16,8 @@ def action_mass_mailing_attendees(self):
             'target': 'current',
             'context': {
                 'default_mailing_model_id': self.env.ref('event.model_event_registration').id,
-                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', '!=', 'cancel')])
+                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', '!=', 'cancel')]),
+                'default_subject': f"Event: {self.name}",
```

Comment:
Don't forget the translation (same for others)
```suggestion
                'default_subject': _("Event: %s", self.name),
```

Reply:
Hello @jeh-odoo 

Changes are done.
Just waiting for the runbot.

Thanks!!